_operator_reference/security.md

```diff
@@ -327,7 +327,7 @@ uiSecurity:
 ```
```

Comment:
`authz.groupMembership.google.credentialPath` also supports `encryptedFile...` secret reference.
`authz.groupMembership.github.accessToken` also supports `encrypted...` secret reference.
`authz.groupMembership.ldap.managerPassword` also supports `encrypted...` secret reference.

Reply:
Again, didn't follow all the way into the RoleProvider classes. Sorry!